pkg/splunk/util/messages.go

```diff
@@ -26,4 +26,10 @@ const (
 
 	// Missing Token error
 	missingTokenError = "Couldn't convert to splunk readable format, %s token missing"
+
+	// nilSecretDataError indicates nil secret data
+	invalidSecretDataError = "Nil secret data"
```

Comment:
nit: is this `nilSecretDataError` or `invalidSecretDataError` ? Comment should be in sync with the function name :)

Reply:
Done